packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -95,9 +95,9 @@ const generateAltButton = (color: string, theme: Record<string, any>) => {
 const $spinnerSize = cssVar('spinner-size');
 
 export const Button: ComponentStyleConfig = {
-  baseStyle: {
-    borderRadius: '0.375rem',
-  },
+  baseStyle: ({ theme }) => ({
+    borderRadius: theme.radii['radii-button'] || '0.375rem',
+  }),
```

Comment:
I think you only need to do this:

```suggestion
  baseStyle: {
    borderRadius: 'radii-button',
  },
```